.circleci/tests/test_reho.py

```diff
@@ -0,0 +1,116 @@
+"""Test for ReHo."""
+import os
+
+import nibabel as nb
+import numpy as np
+
+from xcp_d.utils.write_save import read_ndata, write_ndata
+from xcp_d.workflow.restingstate import init_3d_reho_wf, init_surface_reho_wf
+
+
+def noisy(image):
```

Comment:
Can you rename this function to be more explicit (e.g., `add_noise`). Also, it would be great if it was private (e.g., `_add_noise`).